db/migrate/20251125202538_rebake_posts_with_heading_anchors.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+class RebakePostsWithHeadingAnchors < ActiveRecord::Migration[8.0]
+  def up
+    execute <<~SQL
+      UPDATE posts SET baked_version = NULL
+      WHERE cooked LIKE '%class="anchor"%'
+    SQL
+  end
+
+  def down
+  end
```

Comment:
does it make sense to add `IrreversibleMigration`?

```suggestion
  def down
    raise ActiveRecord::IrreversibleMigration
  end
```